shex/sio-model/disability.shex

```diff
@@ -0,0 +1,72 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v020/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/> 
+PREFIX sio: <http://semanticscience.org/resource/>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX snomedct: <http://purl.bioontology.org/ontology/SNOMEDCT/>
+
+:personShape IRI {
+  a [ sio:SIO_000498 ] ;
+  sio:SIO_000228 @:roleShape ;
+  sio:SIO_000217 @:qualityShape
+}
+
+:roleShape IRI {
+  a [ obo:OBI_0000093 ] ;
+  sio:SIO_000356 @:testprocessShape
+}
+
+:qualityShape IRI {
+  a [ obo:NCIT_C21007 ] ;
+  sio:SIO_000642 @:answerShape ;
+  sio:SIO_000216 @:scoreShape
+}
+
+:questionnaireShape IRI {
+  a [ obo:NCIT_C130322 ] ;
+  sio:SIO_000028 @:questionnaire_questionShape
+}
+
+:questionnaire_questionShape IRI {
```

Comment:
can you use camel case here? Replace `:questionnaire_questionShape` by `:questionnaireQuestionShape`